src/CommonDBTM.php

```diff
@@ -434,6 +434,7 @@ public function getID()
    **/
     public function post_getFromDB()
```

Comment:
You should update all `post_getFromDB` overrides in order to call `parent::post_getFromDB()`, to ensure that this piece of code is always used.
Another solution would be to directly call this method in `CommonDBTM::getFromDB()`, right before the call to `$this->post_getFromDB()`.